UI/Source/CollectionViews/CollectionViewModelDataSource.swift

```diff
@@ -1057,10 +1062,15 @@ extension CollectionViewModelDataSource: NSCollectionViewDataSource {
 
     private func rebindViewAtIndexPath(_ indexPath: IndexPath, toViewModel viewModel: ViewModel) {
         guard let collectionView = collectionView else { return }
-        guard let item = collectionView.item(at: indexPath as IndexPath) as? CollectionViewHostItem else { return }
+        guard let item = collectionView.item(at: indexPath) as? CollectionViewHostItem else { return }
 
         willRebindViewModel(viewModel)
         item.hostedView?.bindToViewModel(viewModel)
+
+        // When rebinding, re-apply attributes so that the updated ViewModel is notified.
```

Comment:
Yeah I was mostly confused with the wording of the comment "so that the updated ViewModel is notified"

Also I'm not sure this is really a good idea, a View that depends on apply getting called again with same attributes every time after binding is a buggy implementation IMO. NSCollectionView doesn't call `apply(layoutAttributes:)` unless the attributes change, so this same situation can happen even without a pilot rebind. Maybe we can sync up offline re: what the problem was in the app this fixes.